Cli/Command/AnalyzeCommand.php

```diff
@@ -27,6 +27,7 @@ protected function configure()
             ->addArgument('project-uuid', InputArgument::REQUIRED)
             ->addOption('format', null, InputOption::VALUE_REQUIRED, 'To output in other formats', 'txt')
             ->addOption('reference', null, InputOption::VALUE_REQUIRED, 'The git reference to analyze')
+            ->addOption('branch', null, InputOption::VALUE_REQUIRED, 'The analysis current branch')
```

Comment:
should it have a `-b` shortcut ?

Reply:
I'd prefer not, I would like to be coherent with other options not having one. You can create a PR to add a shortcut on all option if you want, I'd be happy to merge it :) !